### DISCUSSION TITLE
Some secrets don’t belong in your password manager.

### DISCUSSION THREAD

**Discussion by cottsak** [mprimi/portable-secret#10] [General]
> [Some secrets don’t belong in your password manager](https://mprimi.github.io/portable-secret/#store-top-level-secrets).

can you pls unpack this a little more?

**Comment by mprimi** [comment]
I don't think I'm saying anything outrageous. What I meant is:

there are some secrets (say, my tax return from last year), for which I gain nothing by adding them to a password manager: I don't need access to this document on the go from multiple devices.

So adding it to the password manager is just one more way it could leak, without extra benefit.

Hence, I don't add my tax return to the password manager.

Instead, I just chose to keep it (encrypted at rest) on some subset of my devices. And backed up with the rest of my files.

**Reply by cottsak** [reply]
I guess that makes sense. Maybe I often associate “secret” with “credential” especially in the context of a password manager. I wouldn’t normally use the example of a tax return when it think of “secrets”.
